.github/workflows/weekly_crowdin_sync.yml

```diff
@@ -0,0 +1,31 @@
+name: Sync localisations from Crowdin
+on:
+  schedule:
+    - cron: '0 0 * * 6'
+
+jobs:
+  update-publicsuffix-data:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+
+      - name: Download new publicsuffix data
```

Comment:
This needs to be updated. Otherwise this looks good and is pretty useful, thanks!

Reply:
Done, thanks!